backend/controller/sql/schema/20240926231955_timeline_async_call_event_types.sql

```diff
@@ -0,0 +1,6 @@
+-- migrate:up
+
+ALTER TYPE event_type ADD VALUE IF NOT EXISTS 'cron_scheduled';
```

Comment:
let's chat about what these events will have. One option would be to have a `cron` event type and use the custom fields and payload for the `scheduled` part. This works assuming the payloads of the different cron types would be the same or similar. If each `cron` event has significantly different payloads, then `cron_scheduled` makes sense.